COLLABORATOR_GUIDE.md

```diff
@@ -24,8 +24,10 @@ understand the project governance model as outlined in
 
 ## Issues and Pull Requests
 
-Courtesy should always be shown to individuals submitting issues and
+Courtesy should **always** be shown to individuals submitting issues and
```

Comment:
Not an objection, but just a comment that I try to be very sparing with bold, italics, etc. It gets out of hand pretty easily.